heroku/resources.py

```diff
@@ -0,0 +1,32 @@
+
+from conftest import heroku_client
+
+
+def users_no_2fa():
```

Comment:
The pattern that we have been using so far is to get "all users" and then test whether or not they have 2fa within the `test_*` function. This allows for you to see the number of passing as well, which can be quite useful.

Reply:
Agreed for new code. I'm putting that as "phase 2" for a few reasons:
1. current goal is to get existing code running in framework.
2. that approach pretty much needs caching working, which for some reason it isn't (that's also a phase 2)
3. As more tests get added (desired, but low priority), the work will happen.

I've added #115 so we don't forget this :)